genesis/process/argGenesisBlockCreator.go

```diff
@@ -71,4 +71,5 @@ type ArgsGenesisBlockCreator struct {
 	importHandler update.ImportHandler
 
 	ShardCoordinatorFactory sharding.ShardCoordinatorFactory
+	MapDNSV2Addresses       []string
```

Comment:
this is a list. and in case of DNSV2 we have only 1 address.

Reply:
Yes, but from config we get []string, so I kept it like that for consistency, although it will only contain 1 address

Reply:
just delete the "Map" prefix